prerequisites/build-functions/edit_GCC_download_prereqs_file_if_necessary.sh

```diff
@@ -14,7 +14,7 @@ replace_wget()
     sed -i${backup_extension} -e '2 a\'$'\n'". ${OPENCOARRAYS_SRC_DIR}/prerequisites/build-functions/ftp_url.sh"$'\n' "${download_prereqs_file}"
```

Comment:
I think this sed command may be unnecessarily complex and possibly erroneous, from my understanding of sed and the append command, which mostly comes from reading this: http://www.grymoire.com/Unix/Sed.html#uh-40, but I am hesitant to change it without knowing a  good way to test this. Also, shellcheck complains about `\'` and says the correct way to escape single quotes is `'''` but this expression is so complex I'm not confident that that is what we are actually trying to do here.